modules/labs-react/combobox/stories/stories.tsx

```diff
@@ -38,10 +38,12 @@ const groupOfResults = (
   items: simpleAutoComplete(count, showDisabledItems, 10),
 });
 
-export const Autocomplete: FC<Omit<ComboboxProps, 'children'> & {
-  group?: boolean;
-  showDisabledItems?: boolean;
-}> = ({showClearButton, group, showDisabledItems = false, ...props}) => {
+export const Autocomplete: FC<React.PropsWithChildren<
+  Omit<ComboboxProps, 'children'> & {
+    group?: boolean;
+    showDisabledItems?: boolean;
+  }
+>> = ({showClearButton, group, showDisabledItems = false, ...props}) => {
```

Comment:
This is kind of messy... In CK7, `children` was added to the props interface automatically (in React 18 it was removed... so we went a different direction). BUT, the CK version of `children` allows you to override `children`. If your prop interface includes `children`, yours will be uses instead of the default `children: React.ReactNode`.

This is more of an FYI and not related to this specific change since `Autocomplete` doesn't use our createComponent utility functions. This type of messy typing is because `Combobox` restricts `children` where most of our compound components do not do this.